src/core/reducers/installations.js

```diff
@@ -13,10 +14,12 @@ import {
   UNINSTALL_COMPLETE,
   acceptedInstallTypes,
 } from 'core/constants';
+import log from 'core/logger';
 
 
 export default function installations(state = {}, { type, payload }) {
   if (!acceptedInstallTypes.includes(type)) {
+    log.info(`${type} not in acceptedInstallTypes so action is being ignored.`);
```

Comment:
I added this because this was totally unexpected to me and it tripped me up for a good thirty minutes writing this patch 😢 
Hopefully this will prevent anyone else from getting confused by this bit of code... though I'm wondering why it's there at all...